handsontable/src/pluginHooks.js

```diff
@@ -297,34 +298,48 @@ const REGISTERED_HOOKS = [
   'afterInit',
 
   /**
-   * Fired after new data is loaded (by `loadData` or `updateSettings` method) into the data source array.
+   * Fired after the [`loadData()`](@/api/core.md#loaddata) method,
+   * or the [`updateSettings()`](@/api/core.md#updatesettings) method,
+   * modifies Handsontable's [`data`](@/api/options.md#data).
```

Comment:
@warpech
Re: 1.
Generally, it's good practice to avoid past tense and passive voice (unless necessary). But we don't have style guide rules set for such descriptions yet, so, for consistency with the other hook descriptions, I'd go for:
    ```
    Fired after Handsontable's `data` gets modified by the `loadData()` method or the `updateSettings()` method.
    ```

Re: 2.
> In the description of updateSettings() I can't see that it fires afterSetData, but the name of that method is mentioned here

As I understand it, `updateSettings()` fires the `afterLoadData` and `beforeLoadData` hooks **only** when it modifies the `data` configuration option (@jansiegel pls let me know if I'm wrong) – that's why those hooks were not listed under `updateSettings()` before. We can add them, but wouldn't it affect other hooks as well? (e.g. if you use `updateSettings()` to change the `collapsibleColumns` option, it would fire the `afterColumnCollapse`, right?)

Re: 3.
As discussed, I removed `setData()` from the docs.